doc/sphinx/proof-engine/tactics.rst

```diff
@@ -74,29 +74,31 @@ specified, the default selector is used.
 Bindings list
 ~~~~~~~~~~~~~~~~~~~
 
-Tactics that take a term as an argument may also support a bindings list,
-so as to instantiate some parameters of the term by name or position.
-The general form of a term equipped with a bindings list is ``term with
-bindings_list`` where ``bindings_list`` may be of two different forms:
+Tactics that take a term as an argument may also support a bindings list
+to instantiate some parameters of the term by name or position.
+The general form of a term with a bindings list is
+:n:`@term with @bindings_list` where :token:`bindings_list` can take two different forms:
 
 .. _bindings_list_grammar:
 
-  .. productionlist:: `bindings_list`
+  .. productionlist:: bindings_list
+     ref           : ident
+                   : INT
```

Comment:
Change this in:

```rst
  .. productionlist:: bindings_list
     ref           : `ident`
                   : `num`
```

`num` and `ident` are both already defined grammar tokens.